packages/styles/scss/components/image-with-caption/image-with-caption.scss

```diff
@@ -9,13 +9,13 @@
 
 @mixin image-with-caption {
   .#{$prefix}--image-with-caption {
-    margin-top: $carbon--spacing-07;
+    margin-top: $carbon--spacing-06;
```

Comment:
did we need to change the margins for the `ImageWithCaption` component? I thought we were just changing the styling in `ContentItem`

Reply:
I did it to match the match the specs that Lara mentioned above.

Reply:
@guilhermelMoraes I believe Lara was reminding us to remove the changes you had initially made to the `Content Group` `margin-top` when you first sent the PR. 

I was also under the impression we didn't need to change the `ImageWithCaption` component. However, the most recent visual specs for media is 32px above and below, so not sure if we want to fix that here or later

Reply:
if that's the case I would revert back to `margin-top: to $carbon--spacing-07`

Reply:
Let's leave ImageWithCaption alone for now. There's work coming up with the Media manager that will be addressing padding there.

Reply:
Ok got it. So I'll keep the changes only in `ContentItem` like we discussed previously.